menyer/hellskitchen/style.css

```diff
@@ -0,0 +1,31 @@
+body{
+    background-color: bisque;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+header{
+    background-color: deeppink;
+}
+
+header,
+main,
+footer{
+    text-align: center;
+    margin: auto;
+    max-width: 700px;
+}
+ul{
+    list-style: none;
+    padding: 0%;
+    margin: 20px;
+}
+li{
+    font-size: large;
+}
+a:link{
+    text-decoration: none;
```

Comment:
Usually, you have one empty line between each selector.

Reply:
okej